CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [2.7.2] - 2025-03-28
+
+* Fixed
+  * Retrieve disk info differently based on OTP version
+  * Emit a warning when trying to retrieve public certs through Erlang on < OTP 25
+  * Always run 'mix compile --warnings-as-errors' in CI
+
+* Updated
+  * whenwhere 0.1.1 => 0.1.2
+  * nerves_runtime 0.13.7 => 0.13.8
```

Comment:
I don't include changes to `mix.lock` files since they aren't shipped in the hex package, so the `nerves_runtime` version bump doesn't need to be included. The `whenwhere` bump is interesting since the `mix.exs` was updated to force the version bump. That means that anyone updating NHL will get the version bump whether they like it or not. Perhaps reword this to "Force whenwhere 0.1.2 to fix compatibility with OTP 24 and earlier".

Reply:
Makes sense! Fixed.